x-pack/plugins/canvas/canvas_plugin_src/elements/index.ts

```diff
@@ -62,6 +63,7 @@ const elementSpecs = [
   verticalProgressBar,
   verticalProgressPill,
   tagCloud,
+  metricVis,
```

Comment:
Can we hold this out for now. We can follow up in another PR to add it in and name it appropriately.

Reply:
Done.